app/views/placements/schools/placements/preview.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= content_for :page_title, sanitize(@placement.title) %>
+
```

Comment:
The info panel content will be _slightly different_ here I think. We need to check with @petejobeskinandcarta.